extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -270,6 +274,8 @@ export class ProjectsController {
 	public async publishToDockerContainer(context: Project | dataworkspace.WorkspaceTreeItem, deployProfile: IDeployProfile): Promise<void> {
 		const project: Project = this.getProjectFromContext(context);
 		try {
+			TelemetryReporter.sendActionEvent(TelemetryViews.ProjectController, TelemetryActions.publishToContainer);
```

Comment:
@llali I can remove the publish to container ones if you were planning on adding telemetry for it

Reply:
it's up to you. If you remove it, I' send a PR to add it back. I' thinking to keep it if you already add it but if adding too much work for you (testing verifying ...) I can add it in mine

Reply:
ok I'll keep it for now, and you can add to it later if there are additional things you wanna add